pkg/commands/git.go

```diff
@@ -499,26 +499,22 @@ func (c *GitCommand) Ignore(filename string) error {
 }
 
 // Show shows the diff of a commit
-func (c *GitCommand) Show(sha string) string {
-	result, err := c.OSCommand.RunCommandWithOutput("git show --color " + sha)
-	if err != nil {
-		panic(err)
-	}
-	return result
+func (c *GitCommand) Show(sha string) (string, error) {
+	return c.OSCommand.RunCommandWithOutput(fmt.Sprintf("git show --color %s", sha))
 }
 
 // Diff returns the diff of a file
 func (c *GitCommand) Diff(file File) string {
 	cachedArg := ""
+	trackedArg := "--"
```

Comment:
In my opinion it's easier to see variables define in the function at the top, it's your initial state, you know that this is what's in use in the whole function and if you don't go through any condition the value is what you will get. If you have many variable and a long function it could be hard to check where a variable is defined to get its initial value, here you know everything is at the top so it ease the look up.